cms/djangoapps/contentstore/management/commands/tests/test_cleanup_assets.py

```diff
@@ -21,13 +24,19 @@ class ExportAllCourses(ModuleStoreTestCase):
     """
     Tests assets cleanup for all courses.
     """
+    course_dir = TEST_DATA_DIR / "course_ignore"
+
     def setUp(self):
         """ Common setup. """
         super(ExportAllCourses, self).setUp()
-
         self.content_store = contentstore()
         # pylint: disable=protected-access
         self.module_store = modulestore()._get_modulestore_by_type(ModuleStoreEnum.Type.mongo)
+        add_temp_files_from_dict(DOT_FILES_DICT, self.course_dir / "static")
+
+    def tearDown(self):
+        super(ExportAllCourses, self).tearDown()
+        remove_temp_files_from_list(DOT_FILES_DICT.keys(), self.course_dir / "static")
```

Comment:
I recently listened to a podcast which included advice to always use addCleanup (in setUp) instead of tearDown, because the latter isn't called if there's an error in setUp. It also lets you use multiple cleanup functions, so an error in tearDown won't prevent other cleanup work from happening.